README.md

```diff
@@ -227,75 +227,109 @@ _\* These are the default mobility profile values._
+application has a format of `<deviceName>_<antennaId>` e.g.
+`Reader-10-EF-25_1` where `Reader-10-EF-25` is the deviceName and `1` is the antennaId.
+ 
+To get the list of LLRP devices or readers connected,
+`GET` to the `/api/v1/readers` endpoint:   
+                                                       
+    curl -o- localhost:48086/api/v1/readers
     
-- User needs to configure the alias using Consul. This can be achieved via Consul’s UI or CLI
-  - **Setting Alias via Consul UI**
-    - Create a folder named `Aliases` under [Edgex Consul][consul_root] and
-      add Key Value pairs.
-        
-      ![Creating Aliases folder](images/consul_alias_folder.png)
-    - Key is the default alias which is `<deviceName>_<antennaId>`. The value must be the alias value. Examples of KV pairs:
-         - Speedway-10-EF-25_1: Freezer
-         - Speedway-10-EF-25_2: Backstock
-         
-      ![Adding KV pairs](images/consul_kv_pairs.png)
-      
-      ![Aliases created](images/aliases.png)
-           
-    - Everytime the user creates/updates the Aliases folder the configuration changes apply to the application dynamically, and the updated alias can be seen under tag location `(location_alias)`
-      
-      `GET` to the `/api/v1/inventory/snapshot` endpoint:   
-                                                               
-            curl -o- localhost:48086/api/v1/inventory/snapshot
-            
-      ```json
-         [
-           {
-             "epc": "30143639f8419145db602154",
-             "tid": "",
-             "location": {
-               "device_name": "SpeedwayR-10-EF-25",
-               "antenna_id": 1
-             },
-             "location_alias": "Freezer",
-             "last_read": 1601441311411,
-             "last_arrived": 1601441265669,
-             "last_departed": 0,
-             "state": "Present",
-             "stats_map": {
-               "SpeedwayR-10-EF-25_1": {
-                 "last_read": 1601441311411,
-                 "mean_rssi": -54.25
-               }
-             }
-           }
-         ]     
-      ```
-            
-            
+```json
+{
+  "Readers": [
+    "SpeedwayR-10-EF-25"
+  ]
+}
+```
+
+### Via configuration.toml (before deployment)
+If you already know they alias values you would like to use before deployment, they can be defined in your
+`configuration.toml` file. There is a section called `[Aliases]` that is defaulted to empty.
+
+In order to override an alias, set the default alias as the key, and the new alias as the value you want, such as:
+
+    [Aliases]
+    Reader-10-EF-25_1 = "Freezer"
+    Reader-10-EF-25_2 = "Backroom"
+
+#### EdgeX Hanoi implementation
+When EdgeX Ireland is released, this service will be updated to use `AppCustom` in order to configure
+Aliases via toml. For now, this is the way the code has been implemented:
+
+##### If `overwriteConfig` _is enabled_ via the command line:
```

Comment:
This `overwriteConfig` the edgex built in `-o, --overwrite` capability, correct?

What you have described here in both sections is would the services do normally without  `-o, --overwrite` specified. 
That is the service will check if config is already in Consul. If it is there it will use it. If not it will push it into Consul.
Please combine these descriptions under header somethin like `Configuration for Normal Service Startup`

When `-o, --overwrite` is use, the service will always pushes the config from TOML to Consul, overriding any values already in Consul every time the service starts. This is really meant for development, so please mention that. Would change heading to something like `Overwrite config in Consul (During Development)`

Reply:
Yes, I was talking about the edgex built in `-o, --overwrite` capability. Keep in mind however that will not affect the Aliases section as it is not part of the go config struct, and will be skipped on overwrite. I will re-write the documentation to make it clearer like you have suggested.

In addition, this was something which had some uncertainty that I wanted to ask about. This is why I specifically called out the logic flow for each case so users would be aware of what to expect in each scenario. `Aliases` is a `map[string]string` as opposed to simple pre-defined key-value pairs. What this means is that the actual keys are arbitrary and not known beforehand.

Imagine Consul already contains the following key/values:
`rfid-llrp-inventory/Aliases/Reader-10-EF-25_1` = `POS`
`rfid-llrp-inventory/Aliases/Reader-20-20-20_1` = `Store`

And the user passes `-o` flag, with the following toml file:

    [Aliases]
    Reader-10-EF-25_1 = "Freezer"
    Reader-10-EF-25_2 = "Backroom"

`Reader-10-EF-25_1` already exists with a value of `POS` and will be overwritten to `Freezer`. `Reader-10-EF-25_2` is new and will be added as expected with value `Backroom`. These are the obvious ones. 

The trouble comes with `Reader-20-20-20_1`. This key is not present in the TOML file, but was pre-existing. This means that there is nothing to "overwrite" but is more a lack of inclusion issue. Should we go and remove that KV or should we leave it alone? Right now I am leaving it alone, but I can add code that removes these corner cases. Removing it would make sense if you want to treat the `Aliases` section as a whole instead of singular key/values.

**Existing PR solution:**
`rfid-llrp-inventory/Aliases/Reader-10-EF-25_1` = `Freezer`
`rfid-llrp-inventory/Aliases/Reader-10-EF-25_2` = `Backroom`
`rfid-llrp-inventory/Aliases/Reader-20-20-20_1` = `Store`

**Potential solution:**
`rfid-llrp-inventory/Aliases/Reader-10-EF-25_1` = `Freezer`
`rfid-llrp-inventory/Aliases/Reader-10-EF-25_2` = `Backroom`

Reply:
Yep, I like all the extra explanation, It was just the first bit that was a bit off. THX!